docs-italia-theme/static/js/theme.js

```diff
@@ -1,174 +1,314 @@
+          $(this).removeClass('active');
+          that.$body.removeClass('no-scroll');
+          $wrap.removeClass('active');
+        }
+      });
+
+      // Open nav ( toch monitor)
+      $expanded.on('click' , function(){
+        var $nav = $(this).closest('.chapter-header').find('.chapter-nav__list--hidden'),
+            $wrap = $(this).closest('.chapter-header').find('.title-wrap'),
+            lineHeight = parseInt($(this).closest('.chapter-header').find('.title-has-nav').css('line-height'))+4;
+        if( $nav.hasClass('active') ) {
+          $nav.removeClass('active');
+          $wrap.removeClass('active');
+        } else {
+          $nav.addClass('active');
+          if( that.$window.outerWidth() > 992 ) {
+            $wrap.addClass('active');
+            $wrap.find('.title__background').css('height',lineHeight);
+            if(that.$window.outerWidth() <= 576) {
+              that.$body.addClass('no-scroll');
             }
+          }
         }
-    };
-
-    nav.onScroll = function () {
-        this.winScroll = false;
-        var newWinPosition = this.win.scrollTop(),
-            winBottom = newWinPosition + this.winHeight,
-            navPosition = this.navBar.scrollTop(),
-            newNavPosition = navPosition + (newWinPosition - this.winPosition);
-        if (newWinPosition < 0 || winBottom > this.docHeight) {
-            return;
+      });
+
+
+      // Close when bosy is clikked.
+      that.$body.on('click' , function(event){
+        var $nav = that.$title.closest('.chapter-header').find('.chapter-nav__list--hidden');
+        if( !$('.chapter-header').has(event.target).length > 0 ) {
+          $('.chapter-nav__list--hidden').removeClass('active');
+          $('.title-wrap').removeClass('active');
         }
-        this.navBar.scrollTop(newNavPosition);
-        this.winPosition = newWinPosition;
-    };
-
-    nav.onResize = function () {
-        this.winResize = false;
-        this.winHeight = this.win.height();
-        this.docHeight = $(document).height();
-    };
-
-    nav.hashChange = function () {
-        this.linkScroll = true;
-        this.win.one('hashchange', function () {
-            this.linkScroll = false;
-        });
-    };
-
-    nav.toggleCurrent = function (elem) {
-        var parent_li = elem.closest('li');
-        parent_li.siblings('li.current').removeClass('current');
-        parent_li.siblings().find('li.current').removeClass('current');
-        parent_li.find('> ul li.current').removeClass('current');
-        parent_li.toggleClass('current');
+      });
+
     }
+  }
+})(jQuery);
 
-    return nav;
-};
+ThemeChapterNav.init();
 
-module.exports.ThemeNav = ThemeNav();
+},{}]},{},["docs-italia-theme"]);
 
-if (typeof(window) != 'undefined') {
-    window.SphinxRtdTheme = {StickyNav: module.exports.ThemeNav};
-}
-},{"jquery":"jquery"}]},{},["docs-italia-theme"]);
+},{}]},{},["docs-italia-theme"]);
```

Comment:
Questo file è già molto grande, rischia di diventare di difficile manutenzione. Suggerirei di mettere in piedi almeno una semplice concatenazione su `grunt` (se non ovviamente un `module bundler`, anche se non è indispensabile su questo progetto), per avere "moduli" separati.